### DISCUSSION TITLE
@loopback/build/config/tsconfig.common.json' not found

### DISCUSSION THREAD

**Discussion by vishnukumarps** [loopbackio/loopback-next#7804] [Q&A] [answered]
![Screenshot (39)](https://user-images.githubusercontent.com/46111882/131275854-476a8865-bcb4-4d07-985e-5ec89ed61575.png)

HI,
I couldn't run the access-control-migration example. I have cloned the repository and run the command npm install
while running the application  I get "@loopback/build/config/tsconfig.common.json' not found" error
Please help me to fix this issue

**Comment by achrinza** [comment, marked as answer]
Please use the LoopBack 4 CLI (`lb4 example`) to download the examples. This will use the production releases of LoopBack 4.

If you need to use unpublished releases via the monorepo, run `npm ci` at the root directory of the monorepo and then `npm start` in the directory of the example you want to execute.

**Reply by vishnukumarps** [reply]
Thank you for replying back to me. I was cloning the entire repo and running it. 
When I use lb4 example access-control-migration it is working fine
